packages/components/src/input/src/Input.css

```diff
@@ -9,17 +9,27 @@
 }
 
 .o-ui-input {
-    display: inline-block;
+    display: inline-flex;
```

Comment:
Using display block makes it harder to deal with icons being non absolutely positioned anymore.

Reply:
Actually upon rereading your comment, I see what you mean, I don't see many downsides, although The official specification states an input element is displayed as inline-block.

Reply:
Awesome then!

Reply:
To make sure we are on the same path since the official spec is inline-block and block applies to children, I would keep inline-flex as how we manage the children's is not of the consumer's business.

Reply:
If the consumer of the component doesn’t have to update his layouting because we use “inline-flex” instead of “inline-block” then it’s fine with me!